rune/tools/rpm/Makefile

```diff
@@ -0,0 +1,9 @@
+.PHONY: rpm
+
+PROTOBUF_VERSION := 1.3.5
+PROTOBUF_TARBALL_URL := https://github.com/golang/protobuf/archive/v${PROTOBUF_VERSION}.tar.gz
+
+rpm:
+	@wget -O /root/rpmbuild/SOURCES/protobuf-v${PROTOBUF_VERSION}.tar.gz $(PROTOBUF_TARBALL_URL)
```

Comment:
the variable used to hold the path to rpmbuild should be export by the top Makefile.